### PR TITLE
[acl] Add default deny rule for l3 table

### PR DESCRIPTION
**Pull request by taoyl-ms**

### CODE REVIEW

File: src/sonic-config-engine/tests/sample_output/rules_for_dataacl.json [hunk 1]
```diff
@@ -34,5 +34,12 @@
             "priority":9996
         },
         "OP":"SET"
+    },
+    {
+        "ACL_RULE_TABLE:dataacl:default_rule":{
+            "PACKET_ACTION":"DROP",
```
Comment (stcheng) [inline src/sonic-config-engine/tests/sample_output/rules_for_dataacl.json]: Add one more attribute: `"ETHER_TYPE":"0x0800",`

File: src/sonic-config-engine/tests/sample_output/rules_for_dataacl.json [hunk 2]
```diff
@@ -34,5 +34,12 @@
...
+            "priority":1
```
Comment (stcheng) [inline src/sonic-config-engine/tests/sample_output/rules_for_dataacl.json]: All capitalized to make it look unified.

Comment (taoyl-ms) [inline src/sonic-config-engine/tests/sample_output/rules_for_dataacl.json]: This is directly from table name in json.

File: src/sonic-config-engine/tests/sample_output/rules_for_dataacl.json (hunk 1 above)
Comment (taoyl-ms) [inline src/sonic-config-engine/tests/sample_output/rules_for_dataacl.json]: Shouldn't none-ip packet be dropped as well?

Comment (lguohan) [inline src/sonic-config-engine/tests/sample_output/rules_for_dataacl.json]: all packet includes lldp, arp, lacp, they should not be dropped. It is better to limit to only ip.

Comment (taoyl-ms) [inline src/sonic-config-engine/tests/sample_output/rules_for_dataacl.json]: Didn't quite get it. Shouldn't we have an explicit ACL rule to allow lldp/arp packets?

Comment (stcheng) [inline src/sonic-config-engine/tests/sample_output/rules_for_dataacl.json]: The [implicit rule](http://www.cisco.com/c/en/us/td/docs/switches/datacenter/nexus3000/sw/security/503_u3_1/b_Cisco_n3k_Security_Config_503_u3_1/b_Cisco_n3k_Security_Config_503_u3_1_chapter_01011.html#con_1300782) is to drop all IP and IPv6 packets.

File: src/sonic-config-engine/tests/sample_output/rules_for_dataacl.json (hunk 2 above)
Comment (taoyl-ms) [inline src/sonic-config-engine/tests/sample_output/rules_for_dataacl.json]: Resolved.

File: src/sonic-config-engine/tests/sample_output/rules_for_dataacl.json (hunk 1 above)
Comment (taoyl-ms) [inline src/sonic-config-engine/tests/sample_output/rules_for_dataacl.json]: Modified so that it drops IP packets only.

**Pull request merged**